manala.apt/vars/main.yml

```diff
@@ -282,17 +282,9 @@ manala_apt_repositories_patterns:
   elasticsearch_7:
     source: deb https://artifacts.elastic.co/packages/7.x/apt stable main
     key: elasticsearch
-  ansible: "{{
-    {
-      'debian': None,
-      'ubuntu': {
-        'source': 'ppa:ansible/ansible'
-      }
-    }[ansible_distribution|lower]
-  }}"
-  # Deprecated
-  ppa_ansible:
-    source: ppa:ansible/ansible
+  ansible:
+    source: deb http://ppa.launchpad.net/ansible/ansible/ubuntu {{ { 'jessie':'wily','stretch':'xenial','buster':'bionic'}[ansible_distribution_release] }} main
```

Comment:
for `jessie` it's `trusty` :)

Reply:
yes, i already fixed that, wily is the last ubuntu revision of jessie (pushed master)